sleepecg/io/__init__.py

```diff
@@ -2,11 +2,14 @@
 
 from .ecg_readers import read_gudb, read_ltdb, read_mitdb
 from .nsrr import download_nsrr, set_nsrr_token
+from .sleep_readers import SleepRecord, read_mesa
 
 __all__ = [
+    'SleepRecord',
```

Comment:
Why does this need to be exposed? Are users expected to instantiate this class directly? If not and this is handled via corresponding loader functions (which is what I believe), I'd remove this from here.

Reply:
True... I'm still not completely clear on what's supposed to be in `__all__`, depending on the file (`__init__.py`/`some_module.py`) 😅. Would you keep exposing `SleepStage` and `SleepRecord` in `io.sleep_readers`? They shouldn't be used by a package user, but will be required by other modules (like e.g. `download_file` in `io.utils`).

Reply:
Then I wouldn't expose it in `__all__`, which I interpret as the public API (but I might be wrong). Internal modules can always import whatever they want from other local modules.

Reply:
Okay, works for me! I'll open a PR to standardize this once the current ones are merged :)